.github/workflows/lint.yml

```diff
@@ -1,15 +1,15 @@
-name: ❄️ Lint
```

Comment:
If you change this name you need to update the cancel workflow, where this string is used

Reply:
Ah I had missed that, just pushed the fix.  Thanks.